images/macos/toolsets/toolset-12.json

```diff
@@ -0,0 +1,164 @@
+{
+    "xcode": {
+        "default": "13.0",
+        "versions": [
+            { "link": "13.0", "version": "13.0.0"}
+        ]
+    },
+    "xamarin": {
+        "vsmac": "latest",
+        "mono-versions": [
+            "6.12.0.125"
+        ],
+        "ios-versions": [
+            "14.20.0.24"
+        ],
+        "mac-versions": [
+            "7.14.0.24"
+        ],
+        "android-versions": [
+            "11.3.0.4"
+        ],
+        "bundle-default": "6_12_9",
+        "bundles": [
+            {
+                "symlink": "6_12_9",
+                "mono":"6.12",
+                "ios": "14.20",
+                "mac": "7.14",
+                "android": "11.3"
+            }
+        ]
+    },
+    "java": {
+        "default": "8",
+        "versions": [
+            "8", "11"
+        ]
+    },
+    "android": {
+        "platform_min_version": "27",
+        "build_tools_min_version": "27.0.0",
+        "extra-list": [
+            "android;m2repository", "google;m2repository", "google;google_play_services", "intel;Hardware_Accelerated_Execution_Manager"
+        ],
+        "addon-list": [],
+        "additional-tools": [
+            "cmake;3.18.1"
+        ],
+        "ndk": {
+            "lts": "21",
+            "latest": "22"
+        }
+    },
+    "powershellModules": [
+        { "name": "Az" },
+        { "name": "MarkdownPS" },
+        { "name": "Pester" },
+        { "name": "PSScriptAnalyzer" }
+    ],
+    "npm": {
+        "global_packages": []
+    },
+    "brew": {
+        "common_packages": [
+            "ant",
+            "aria2",
+            "bazelisk",
+            "carthage",
+            "cmake",
+            "gh",
+            "gnu-tar",
+            "go@1.17",
```

Comment:
Why `go@1.17` is here? Looks like incorrect merge?

Reply:
Done. Thank you.